scripts/index.js

```diff
@@ -1 +1,95 @@
-// add your solutions here
\ No newline at end of file
+// Positive Numbers
+
+function returnsPositiveNumbers(givenArray) {
```

Comment:
the nice thing about `filter` is that it returns an array for you, so you don't have to declare one and push to it. `filter` and `map` return arrays automatically, but `.forEach` does not.